docker-compose.yml

```diff
@@ -168,7 +168,7 @@ services:
   mirror_transformation_grids:
     image: k3rnelpan1c/alpine-wget:latest
     command: |
-      wget --mirror https://cdn.proj.org/ -P /transformation_grids --no-host-directories
+      wget --no-verbose --mirror https://cdn.proj.org/ -P /transformation_grids --no-host-directories
```

Comment:
Does it at least show what we are downloading? If yes, then I am ok, if it is completely silent, it's problematic.

Reply:
It shows what is downloading, and errors too, but is a bit misleading when things work fine (looks like nothing happened). Will move that out of this PR as not critical anyway